test/graphql/ToMagentoMapperTest.js

```diff
@@ -0,0 +1,141 @@
+/*******************************************************************************
```

Comment:
So far we tried to stick to the rule where we are creating 2 tests or 1 test for each file. So for instance if our source file is CustomerMapper we then have one test file CustomerMapperTest (only UT is need it). If our source file is getCustomer we then have 2 test files: getCustomerTest and getCustomerIT. Can we respect this convention with grappql? I don't see the source file for this file.

Reply:
ToMagentoMapper is in src/graphql/lib or do you mean sth else?

Reply:
yes, I suggest to remove `utils/lib` from the `/src/grapql` and move those file in `src/graphql`.

Reply:
@masodu what do you want all the files in the root folder? Jeniffer initially had all the files in the same folder and I told her to create sub-folders to better organise the code. What's the issue with sub-folders?... I think it makes sense to better organise the code.

Reply:
Also a longer-term objective is to separate Magento specific from more "graphQL generic" code, which could be moved at some point to the "commerce-cif-common" repo and a dedicated package. It's a bit too early for that because we would need more experience with another graphQL endpoint like the one in Commercetools, to be able to better test what/if the generic code could also be reused for CT.

Reply:
Personally I think it is confusing because we are not consistent with the way we have designed our actions. This affects also the way we are organising the tests. I would suggest to keep it simple and put everything in the root and then have all the tests for graph ql in the same folder. Also the `ToMagentoMapper` name is not suggestive enough especially if we want to have it in some common part. Can't we just refactor later when we have a better understanding of what can be and should be reused?